jib-core/src/main/java/com/google/cloud/tools/jib/image/Image.java

```diff
@@ -129,24 +130,36 @@
     }
 
     /**
-     * Sets the number of nanoseconds to wait between healthcheck attempts.
+     * Sets the time to wait between healthcheck attempts.
      *
-     * @param healthInterval the number of nanoseconds to wait between healthcheck attempts
+     * @param healthInterval the time duration in the given {@code sourceUnit} to wait between
+     *     healthcheck attempts
+     * @param timeUnit the unit of the {@code healthInterval} argument
      * @return this
      */
-    public Builder<T> setHealthInterval(@Nullable Long healthInterval) {
-      this.healthInterval = healthInterval;
+    public Builder<T> setHealthInterval(@Nullable Long healthInterval, TimeUnit timeUnit) {
```

Comment:
Not sure what the best practice is here but would `Duration` make more sense here? (It's in the `java.time` package as opposed to the `java.util.concurrent` package for `TimeUnit`)

Reply:
@elharo Would you recommend `(long, TimeUnit)` for the signature or `(Duration)`?

Reply:
Yeah I like this more.